save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/sandbox/SandboxController.kt

```diff
@@ -0,0 +1,84 @@
+package com.saveourtool.save.orchestrator.sandbox
+
+import com.saveourtool.save.agent.AgentVersion
+import com.saveourtool.save.agent.TestExecutionDto
+import com.saveourtool.save.domain.FileKey
+import com.saveourtool.save.domain.TestResultDebugInfo
+import com.saveourtool.save.orchestrator.SANDBOX_PROFILE
+import org.springframework.context.annotation.Profile
+import org.springframework.core.io.Resource
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import java.nio.ByteBuffer
+
+typealias ByteBufferFluxResponse = ResponseEntity<Flux<ByteBuffer>>
+typealias StringResponse = ResponseEntity<String>
+
+/**
+ * Sandbox implementation of endpoints which are required for save-agent
+ */
+@Profile(SANDBOX_PROFILE)
+@RestController
+@RequestMapping("/sandbox")
```

Comment:
The resulting path for all endpoints will be like `/sandbox/internal`, shouldn't it be the other way around?

Reply:
it's for now only "/sandbox", but we need to have open api to communicate with `frontend` from `/sandbox` box.
but I would like to extract it to a dedicated rest controller.
Will add `suffix` to current controller